wiki/tfchain/tfchain_dao.md

```diff
@@ -0,0 +1,68 @@
+# TF Chain DAO (decentralised autonomous organisation)
+
+The DAO on TF Chain was created with the intent to give Farmers voting right on chain. The Council (TF Grid Guardians) can create proposals where Farmers can vote upon.
+
+## Council
+
+A council member is a person with slightly elevated privileges, represented by an account on chain. It is intended that this person acts in the interest of the grid/chain. There can be at most 12 council members at once (though this should be configurable in the code). A council member has the following abilities on chain:
+
+- Create a new voting proposal.
+- Link an existing farming policy to a farm, with suggested limits (see farming policy section).
+- Council only vote (nobody except council member can vote for this) to add or remove a member. For the result of this vote, strict majority is used, i.e. vote passes if votes_for > votes_against. There must also be a time limit on the vote, any council member who doesn't vote before this time implicitly abstains. Note that in case of removal, the member to be removed can vote against that (no special logic to prevent this).
+- Set the certification level of a farm (see bellow).
+
+## Certified node verifier
+
+This is an entity (person, group, ...), represented on chain by an address, which can mark an existing node as certified. Adding addresses to this list is done through a regular vote. Any address in this group can mark any node as certified (and unmark a node as well). The Council can propose an entity to be added as a Node Certifier, this will most likely be an entity who sells hardware to expand the TF Grid. A node Certifier can mark a Node as certified so that the Node received additional rewards every minting period.
+
+## Cerfified farms
+
+A Farm is either `NotCertified` or `Gold`. The DAO can propose that a certain farm get's marked as `Gold` if it has a certain farming setup.
```

Comment:
Why is gold used here instead of Certified?

Reply:
It was specced to be named like that